pkg/kubectl/cmd/util/factory.go

```diff
@@ -134,6 +134,8 @@ type Factory struct {
 	PauseObject func(object runtime.Object) (bool, error)
 	// ResumeObject resumes a paused object ie. it will be reconciled by its controller.
 	ResumeObject func(object runtime.Object) (bool, error)
+	// ResolveImage resolves the image names.
```

Comment:
Can you add a comment for why this function exists? You should mention that it's just a passthrough for Kube but others (like we in OpenShift) support various image formats.